content/blog/2025/05/2025-04-25-bridges-update.md

```diff
@@ -0,0 +1,23 @@
++++
+date = "2025-05-14"
+title = "An Update on Bridges"
+
+[taxonomies]
+author = ["Thib"]
+category = ["Bridges"]
++++
+
+Following up on our [Crossroads blog post](/blog/2025/02/crossroads/) we’re giving an update on bridges. The Foundation indeed can’t afford to keep maintaining the bridges anymore and needs to focus its very little resources on its core missions. 
+
+Element manifested their interest in helping us keep the following public instances of the bridges online.
+
+| Instance | Platform | Project |
+| -------- | -------- | ------- |
+| OFTC     | IRC      | [matrix-appservice-irc](https://github.com/matrix-org/matrix-appservice-irc/) |
+| Snoonet  | IRC      | [matrix-appservice-irc](https://github.com/matrix-org/matrix-appservice-irc/) |
+| Slack    | Slack    | [matrix-appservice-slack](https://github.com/matrix-org/matrix-appservice-slack)  |
+| Bifrost  | XMPP     | [matrix-bifrost](https://github.com/matrix-org/matrix-bifrost)  |
+
+We’re grateful to Element for sponsoring the bridges and keeping them online for the thousands of users who rely on them. Since Element already supports us in many areas, we keep actively looking for additional sponsors to help us grow more independent. You can find ways [to support us on our website](https://matrix.org/support/), the most impactful one being to [join us as a member](https://matrix.org/membership/).
```

Comment:
```suggestion
We’re grateful to Element for sponsoring the bridges and keeping them online for the thousands of users who rely on them. Since Element already supports us in many areas, we keep actively looking for additional sponsors to help us grow more independent. You can find ways [to support us on our website](/support/), the most impactful one being to [join us as a member](/membership/).
```

Relative links please :)